appctl/appctl.go

```diff
@@ -17,10 +17,19 @@ import (
 	"github.com/openstack-k8s-operators/dataplane-node-exporter/log"
 )
 
-func Call(method string, args ...string) string {
+func Call(method string, isOvn bool, args ...string) string {
```

Comment:
Could you change this to be more generic:

```go
type appctlDaemon string

const (
        ovsVSwitchd   AppctlDaemon = "ovs-vswitchd"
        ovnController              = "ovn-controller"
)

func call(daemon appctlDaemon, method string, args ...string) string {
        var rundir string
        var err error

        switch daemon {
        case ovsVSwitchd:
                rundir = config.OvsRundir()
        case ovnController:
                rundir = config.OvnRundir()
        default:
               panic(fmt.Errorf("unknown daemon value: %v", daemon))
        }

        pidfile := filepath.Join(rundir, fmt.Sprintf("%s.pid", dameon))

        // ....

        sockpath := filepath.Join(rundir, fmt.Sprintf("%s.%d.ctl", daemon, pid))

        // ....
}

func OvsVSwitchd(method string, args ...string) string {
         return call(ovsVSwitchd, method, args...)
}

func OvnController(method string, args ...string) string {
         return call(ovnController, method, args...)
}
```

Reply:
Done. Thanks for the review!